postgres/src/test/scala/zio/sql/TestContainers.scala

```diff
@@ -0,0 +1,33 @@
+package zio.sql
+
+import com.dimafeng.testcontainers.SingleContainer
+import com.dimafeng.testcontainers.PostgreSQLContainer
+import zio._
+import zio.blocking.{ effectBlocking, Blocking }
+
+object TestContainer {
+
+  def container[C <: SingleContainer[_]: Tag](c: C): ZLayer[Blocking, Nothing, Has[C]] =
+    ZManaged.make {
+      effectBlocking {
+        c.start()
+        c
+      }.orDie
```

Comment:
Might be a good idea to handle the error so we get nicer test failures if the container doesn't start. I don't like seeing fiber failures in test results because they take up so much space in the console. I think the probability of a container not starting is high enough to handle it.